README.md

```diff
@@ -1,2 +1,6 @@
 # product-tour
 An interactive tour which helps onboard new users to Mautic
+
+## Installation
+
+Reload plugins on the plugin page, and the tours will be available. 
```

Comment:
Can we get more info? Like how does it work when a user goes through the training? Will it appear every time they log in? When they refresh page? Can it be disabled?

It would be nice to have a screenshot of how it looks like.

There is also several ways how to install the plugin. Via Git clone, Composer, SFTP, ... I'm not sure everyone knows the way.